recipes/gnu-time/build.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -x
+set -e
+
+# Adopt a Unix-friendly path if we're on Windows (see bld.bat).
+[ -n "$PATH_OVERRIDE" ] && export PATH="$PATH_OVERRIDE"
+
+# On Windows we want $LIBRARY_PREFIX in both "mixed" (C:/Conda/...) and Unix
+# (/c/Conda) forms, but Unix form is often "/" which can cause problems.
+if [ -n "$LIBRARY_PREFIX_M" ] ; then
+    mprefix="$LIBRARY_PREFIX_M"
+    if [ "$LIBRARY_PREFIX_U" = / ] ; then
+        uprefix=""
+    else
+        uprefix="$LIBRARY_PREFIX_U"
+    fi
+else
+    mprefix="$PREFIX"
+    uprefix="$PREFIX"
+fi
+
+# On Windows we need to regenerate the configure scripts.
+if [ -n "$VS_MAJOR" ] ; then
+    am_version=1.15 # keep sync'ed with meta.yaml
+    export ACLOCAL=aclocal-$am_version
+    export AUTOMAKE=automake-$am_version
+    autoreconf_args=(
+        --force
+        --install
+        -I "$mprefix/share/aclocal"
+        -I "$mprefix/mingw-w64/share/aclocal" # note: this is correct for win32 also!
+    )
+    autoreconf "${autoreconf_args[@]}"
+
+    # And we need to add the search path that lets libtool find the
+    # msys2 stub libraries for ws2_32.
+    platlibs=$(cd $(dirname $(gcc --print-prog-name=ld))/../lib && pwd -W)
+    export LDFLAGS="$LDFLAGS -L$platlibs"
+fi
+
+./configure \
+    --prefix=$mprefix \
+    --disable-dependency-tracking \
+    --mandir=$mprefix/share/man \
+    --infodir=$mprefix/share/info \
+    --program-prefix=g \
```

Comment:
>  What is the relevance of time being a builtin?

Just that naively running `time` in the shell won't work because the builtin supersedes the binary. 

> Just that people hard coded calls to time? If so, it seems like patches that use command time should solve the issue and be acceptable for upstream, right? Arguably these are the correct way of solving this problem compared to hard coding a path as it bypasses the builtin. Or is there more to this that I'm missing?

Exactly. It needs `$(which time)`, `command time` or `/usr/bin/env time`.